rockcraft/models/project.py

```diff
@@ -317,6 +291,16 @@ class Config(CraftBaseConfig):  # pylint: disable=too-few-public-methods
         allow_mutation = False
         extra = pydantic.Extra.forbid
 
+    @override
+    @classmethod
+    def _providers_base(cls, base: str | None) -> bases.BaseAlias | None:
+        """Get a BaseAlias from rockcraft's base."""
+        if not base or base == "bare":
```

Comment:
will also need to handle `devel` here, if this is called with `build-base`

Reply:
Done